examples/nextjs-with-typescript/src/Link.tsx

```diff
@@ -1,9 +1,12 @@
-/* eslint-disable jsx-a11y/anchor-has-content */
 import * as React from 'react';
 import clsx from 'clsx';
 import { useRouter } from 'next/router';
 import NextLink, { LinkProps as NextLinkProps } from 'next/link';
 import MuiLink, { LinkProps as MuiLinkProps } from '@mui/material/Link';
+import { styled } from '@mui/material/styles';
+
+// Add support for the sx prop for consistency with the other branches.
+const Anchor = styled('a')({});
```

Comment:
Please keep the example minimal. There is no need to add another learning layer. Consistency is not a target.

Reply:
It's meant as a bug fix. This style allows to get the `sx` prop no matter the params (using external links or not). I believe Jun added it because it noticed that it wasn't always present.

Reply:
I have created to illustrate it https://codesandbox.io/s/polished-wood-31j6t?file=/pages/index.js